app/models/access_control.rb

```diff
@@ -102,6 +105,10 @@ class AccessControl < ApplicationRecord
     where(id: ids)
   end
 
+  scope :visible_to, ->(_user) do
+    return all
```

Comment:
Should be sufficient:

```suggestion
   all
```